docs/modules/ROOT/pages/configure-cmos.adoc

```diff
@@ -0,0 +1,230 @@
+= Configuration Options
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+.Microlith configuration image
+image::microlith-runtime.png[]
+
+CMOS is designed as a microlith container with several services included in its stack.
+CMOS allows users to control these services in a fine grained manner.
+As shown in the above diagram, you can customize the installation by overriding the default service configuration values using environment variables or by providing a native configuration file for each service.
+
+== Disabling a Service
+
+All the services of CMOS stack are enabled by default.
+If you choose to disable a service, this is achieved by setting the corresponding environment variables of the docker container. The naming convention to be followed to disable a service is `DISABLE_X`, where `X` is the name of the service (e.g.,`DISABLE_ALERT_MANAGER`, `DISABLE_GRAFANA`, `DISABLE_JAEGER`, `DISABLE_LOKI`, `DISABLE_PROMETHEUS` and `DISABLE_WEBSERVER`).
+
+== Logging
+
+By default, logs can be either provided to standard output `(stdout)` or to service-based log files by setting the `ENABLE_LOG_TO_FILE` configuration value.
+If file output is used for the logs, these files will appear in the `/logs/` directory.
+Further, if desired, these logs can externally be shared via a volume or bind mount.
+
+To mount files into a container, use the `docker -v <source>:<destination>` command, ideally as read-only.
+For further information, refer to the official documentation:
+- https://docs.docker.com/storage/volumes/
+- https://docs.docker.com/storage/bind-mounts/
+
+== Cluster Monitor
+
+The cluster manager service of the CMOS stack will auto-start and self-configure itself to monitor a Couchbase cluster using the configuration supplied through the following environment variables:
+
+- `CLUSTER_MONITOR_USER=${CLUSTER_MONITOR_USER:-admin}`: The username for the Couchbase cluster to monitor.
+- `CLUSTER_MONITOR_PWD=${CLUSTER_MONITOR_PWD:-password}`: The password for the Couchbase cluster to monitor.
+- `CLUSTER_MONITOR_ENDPOINT=${CLUSTER_MONITOR_ENDPOINT:-http://localhost:7196}`: The endpoint of the Couchbase cluster to monitor.
```

Comment:
These aren't for the Couchbase server, but for the Cluster Monitor itself - the first two are the credentials for its REST API. Couchbase Server is added through the Add Cluster form, not through environment variables.

I think I know where the confusion stems from - were you perhaps looking at https://github.com/couchbaselabs/observability/blob/main/docs/modules/ROOT/pages/deployment-onpremise.adoc? Because there those variables aren't actually passed to Cluster Monitor itself, but used in the `curl` command at the end:

```
CLUSTER_MONITOR_USER=admin
CLUSTER_MONITOR_PWD=password
CLUSTER_MONITOR_ENDPOINT=http://localhost:$(docker container port couchbase-grafana 7196)
COUCHBASE_USER=Administrator
COUCHBASE_PWD=password
COUCHBASE_ENDPOINT=http://db2:8091

# This is where the variables are used - not in the CMOS configuration
curl -u "${CLUSTER_MONITOR_USER}:${CLUSTER_MONITOR_PWD}" -X POST -d '{ "user": "'"${COUCHBASE_USER}"'", "password": "'"${COUCHBASE_PWD}"'", "host": "'"${COUCHBASE_ENDPOINT}"'" }' "${CLUSTER_MONITOR_ENDPOINT}/api/v1/clusters"
```